frontend/editor.css

```diff
@@ -1606,6 +1609,15 @@ pluto-input:focus-within .cm-s-default .CodeMirror-matchingbracket {
     border-radius: 2px;
 }
 
+pluto-notebook .CodeMirror pre.CodeMirror-line,
+pluto-notebook .CodeMirror pre.CodeMirror-line-like {
```

Comment:
use the same style selector as all other codemirror rules, can we just put this in 

https://github.com/fonsp/Pluto.jl/blob/v0.12.4/frontend/editor.css#L1467

?

Reply:
No, the rule in line 1467 does _not_ override the rule that enables ligatures, so we cannot put the code in the same codeblock, but to the spirit of your comment I added this right below it (probably that's what you meant all along):

![image](https://user-images.githubusercontent.com/8681967/96844919-c5044100-1458-11eb-8266-e827119e5408.png)

and moved the operator to along its friends further below:
![image](https://user-images.githubusercontent.com/8681967/96844986-da796b00-1458-11eb-9e0a-63d2ec493efb.png)

Hope that helps.

btw, I noticed that you use codemirror's _theme selector_ `cm-s-default` everywhere - so that is actually a theme customization. Of course that makes perfect sense for colors, paddings etc

I'm just writing that because when/if someone changes the theme in the future (as DarkMode seems to do) these rules will stop working and the ligatures will be enabled.